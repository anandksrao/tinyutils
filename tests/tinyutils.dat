7
9
3
12.2
0
12
9
4
