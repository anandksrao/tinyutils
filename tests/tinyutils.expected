boolify
1
1
1
1
0
1
1
1
cumsum
7
16
19
31.2
31.2
43.2
52.2
56.2
diffs
2
-6
9.2
-12.2
12
-3
-5
exp
1096.63
8103.08
20.0855
198789
1
162755
8103.08
54.5982
exp10
10000000
1000000000
1000
1.58489e+12
1
1000000000000
1000000000
10000
exp2
128
512
8
4705.07
1
4096
512
16
hist
0	1
1	0
2	0
3	1
4	1
5	0
6	0
7	1
8	0
9	2
10	0
11	0
12	2
histbin
0	6
10	2
log
1.94591
2.19722
1.09861
2.50144
-inf
2.48491
2.19722
1.38629
log10
0.845098
0.954243
0.477121
1.08636
-inf
1.07918
0.954243
0.60206
log2
2.80735
3.16993
1.58496
3.60881
-inf
3.58496
3.16993
2
max
12.2
mean
7.025
median
8
min
0
range
0	12.2
stripfilt
9
3
12.2
0
12
9
4
sum
56.2
ncol
1
1
1
1
1
1
1
1
len
1
1
1
4
1
2
1
1
showcol
1:7
mult mult=2
14
18
6
24.4
0
24
18
8
div div=2
3.5
4.5
1.5
6.1
0
6
4.5
2
round digits=0
7
9
3
12
0
12
9
4
table | sort -k1,1 -n
0	1
3	1
4	1
7	1
9	2
12	1
12.2	1
stripfilt inverse=1
7
stripfilt header=0 inverse=1
stripfilt inverse=1 inverse_early=0 header=0
inrange min=1 max=8
7
3
4
inrange abs=4
3
0
4
